testsuite/MDAnalysisTests/topology/test_guessers.py

```diff
@@ -104,6 +104,9 @@ def test_guess_atom_element_1H(self):
         ('zn', 'ZN'),
         ('Ca2+', 'CA'),
         ('CA', 'C'),
+        ('N0A', 'N'),
```

Comment:
One more nit sorry could you test  Na+ -> Na?

Reply:
I just added two tests for Na+ and Cu2+